lib/standard/stream.nit

```diff
@@ -42,8 +42,11 @@ end
 # A `Stream` that can be read from
 abstract class Reader
 	super Stream
-	# Read a character. Return its ASCII value, -1 on EOF or timeout
-	fun read_char: Int is abstract
+	# Read a character. Returns `null` on EOF or timeout
+	fun read_char: nullable Char is abstract
+
+	# Reads a byte, returns `null` if EOF or timeout
```

Comment:
Both comments (with the top one) are pretty indentical, yet written in a totaly differtent maner.